README.md

```diff
@@ -20,3 +21,25 @@ randomly without memory instead of systematically trying them?
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+The runtime complexity of my version of Brute Sort is $\Theta (n * n!)$. As my algorithm
+is implemented with some for loops we can analyze the complexity using loop logic.
+The first loop will run n times where n is the length of the array, as seen in the loop
+conditions. Then it will call the permutation function which will be called using a starting
+index that is one greater. This results in something reminicent of n(n-1)(n-2)... which is
+n!. Then a second loop is called inside the for loop which runs over the whole array to see
+if it is sorted so it has a runtime of n. So the total runtime can be calculated by 
+multiplying these together resulting in the runtime complexity is $\Theta (n*n!)$.
+
+The best case input would be an array that is already sorted as the program would only
+have to find one permutation of the array. This means that it would only have to run
+the first for loop once and the second one once as well, so the overall runtime would
+be $\Theta (n^2)$
```

Comment:
That seems unnecessarily inefficient. Shouldn't it just have to check whether the array is sorted then?